shuup/xtheme/admin_module/__init__.py

```diff
@@ -96,3 +98,45 @@ def get_notifications(self, request):
                         title=_("Theming"),
                         url="shuup_admin:xtheme.config"
                     )
+
+
+class XthemeSnippetsAdminModule(AdminModule):
```

Comment:
Make this available from theme configuration (action button over table) and the front toolbar.